hello-world/simponic/Makefile

```diff
@@ -0,0 +1,23 @@
+.DEFAULT_GOAL := os.iso
+
+OBJECTS=entry.o kernel.o
+KERNEL_BUILD_FLAGS=-std=c99 -ffreestanding -O0 -Wall -Wextra
+OS_BUILD_FLAGS=-ffreestanding -O2 -nostdlib -lgcc
+
+entry.o: boot.asm 
+	nasm -felf32 boot.asm -o entry.o
+
+kernel.o: kernel.c
+	i386-elf-gcc -c kernel.c -o kernel.o $(KERNEL_BUILD_FLAGS)
+
+os.bin: kernel.o entry.o
+	i386-elf-gcc -T linker.ld -o os.bin $(OS_BUILD_FLAGS) $(OBJECTS)
+
+os.iso: os.bin
+	mkdir -p iso/boot/grub
+	cp os.bin iso/boot/os.bin
+	printf "menuentry 'HelloOS' {\n  multiboot /boot/os.bin\n  set gfxpayload=text\n}" > iso/boot/grub/grub.cfg
+	grub-mkrescue -o os.iso iso
+
+clean:
+	rm -rf iso *.o *.bin
```

Comment:
It would help to add a `run` target here for `qemu-system-i386`

I know you don't run with any special flags, but that would not only facilitate running your demo (`make run`), but also explicitly specify that you expect this to run without any additional flags to the emulator.